cfme/services/catalogs/myservice.py

```diff
@@ -160,3 +163,9 @@ def check_vm_add(self, add_vm_name):
     def reconfigure_service(self):
         sel.force_navigate('reconfigure_service',
                            context={'service_name': self.service_name})
+
+    def test_download_file(self):
+        sel.force_navigate('all_service')
+        download_btn('Download as Text')
+        download_btn('Download as CSV')
+        download_btn('Download as PDF')
```

Comment:
Agreed on the naming here. Plus I take it we are not "checking" that hte file is downloaded OK right? We are just checking the link doesn't blow up?